src/components/ColorPicker/ColorPicker.st.css

```diff
@@ -11,8 +11,15 @@
 }
 
 .root {
-    display: inline-flex;
+    display: inline-block;
     margin: -4px;
+    border: 0;
+    padding: 0.01em 0 0 0;
+    min-width: 0;
```

Comment:
What are these two for?

Reply:
This is a reset for fieldset.
https://gist.github.com/ultrox/84935bf1ce4fc938f61c2142b9a9c1f2